client/src/components/_common/Form/FileInputDropZone.jsx

```diff
@@ -74,6 +74,14 @@ function FileInputDropZone({
                   <InlineMessage type="error">
                     Exceeds File Size Limit
                   </InlineMessage>
+                  <Button
+                    type="link"
+                    onClick={() => {
+                      setRejectedFiles([]);
```

Comment:
This will actually remove all rejected files. I think that actually works, but the way the UI is presented, this should remove only the file you click "Remove" on. You can use `setRejectedFiles` to pass an updated array that excludes the element for the row you're rejecting

Reply:
I'll modify this to remove only the selected oversized file.

Reply:
My most recent push now only removes the oversized file that the user selects.